pkg/epp/requestcontrol/director.go

```diff
@@ -121,7 +122,7 @@ func (d *Director) HandleRequest(ctx context.Context, reqCtx *handlers.RequestCo
 	reqCtx.SchedulingRequest = &schedulingtypes.LLMRequest{
 		RequestId:   reqCtx.Request.Headers[requtil.RequestIdHeaderKey],
 		TargetModel: reqCtx.TargetModelName,
-		Prompt:      prompt,
+		Data:        requestData,
```

Comment:
nit: data is a generic name, is there a more specific name we can use here? should we use body instead since we have headers below?

Reply:
`Input`?

Reply:
isn't this the body but structured?

Reply:
In LLMRequest, it is the only part of the body being captured, so I think Body is appropriate here; I think the type `LLMRequestData` could be named `Body` as well; if we want to pass the raw body in the future, we can add a `RawBody` field later